tests/baselines/reference/parseInvalidNonNullableTypes.errors.txt

```diff
@@ -0,0 +1,53 @@
+tests/cases/compiler/parseInvalidNonNullableTypes.ts(1,30): error TS17019: Adding a postfix '!' to the end of a type is not valid TypeScript syntax. Did you mean to write 'string'?
+tests/cases/compiler/parseInvalidNonNullableTypes.ts(5,30): error TS17020: Adding a prefix '!' to the start of a type is not valid TypeScript syntax. Did you mean to write 'string'?
+tests/cases/compiler/parseInvalidNonNullableTypes.ts(9,16): error TS17019: Adding a postfix '!' to the end of a type is not valid TypeScript syntax. Did you mean to write 'string'?
+tests/cases/compiler/parseInvalidNonNullableTypes.ts(10,16): error TS17019: Adding a postfix '!' to the end of a type is not valid TypeScript syntax. Did you mean to write 'number'?
+tests/cases/compiler/parseInvalidNonNullableTypes.ts(12,16): error TS17020: Adding a prefix '!' to the start of a type is not valid TypeScript syntax. Did you mean to write 'string'?
+tests/cases/compiler/parseInvalidNonNullableTypes.ts(13,16): error TS17020: Adding a prefix '!' to the start of a type is not valid TypeScript syntax. Did you mean to write 'number'?
+tests/cases/compiler/parseInvalidNonNullableTypes.ts(15,16): error TS17019: Adding a postfix '!' to the end of a type is not valid TypeScript syntax. Did you mean to write 'any'?
+tests/cases/compiler/parseInvalidNonNullableTypes.ts(16,10): error TS17019: Adding a postfix '!' to the end of a type is not valid TypeScript syntax. Did you mean to write 'number'?
+tests/cases/compiler/parseInvalidNonNullableTypes.ts(18,16): error TS17020: Adding a prefix '!' to the start of a type is not valid TypeScript syntax. Did you mean to write 'any'?
+tests/cases/compiler/parseInvalidNonNullableTypes.ts(19,10): error TS17020: Adding a prefix '!' to the start of a type is not valid TypeScript syntax. Did you mean to write 'number'?
+
+
+==== tests/cases/compiler/parseInvalidNonNullableTypes.ts (10 errors) ====
+    function f1(a: string): a is string! {

```

Comment:
after reading these examples, I think the only useful place to suggest `NonNullable<T>` would be when `T` is a type parameter. Which is likely rare enough not to be worthwhile.